src/command_modules/azure-cli-lab/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.0.19
```

Comment:
In addition to the history file, please also update the version in the `azure-cli-lab` package's `setup.py`.

Reply:
resolved